### DISCUSSION TITLE
linkerinfo_of函数调用存在拼写错误

### DISCUSSION THREAD

**Discussion by hyyq** [xmake-io/xmake#3764] [General]
https://github.com/xmake-io/xmake/blob/48bea0307854d0ecdaa39e3e4d5a770e4f4cdd48/xmake/core/sandbox/modules/import/core/language/language.lua#L132
...r/core/sandbox/modules/import/core/language/language.lua:132: field 'linkerinfo_of' is not callable (a nil value)

**Comment by waruqi** [comment]
修了，话说你怎么会用到这个接口？自己调用的？

xmake update -s dev

**Reply by hyyq** [reply]
只能说发现的很偶然，最近在研究xmake的实现，很难想象这是一个个人项目，说起来用lua开发的大型项目真的很难阅读😂
